osu.Game/IO/Serialization/IJsonSerializable.cs

```diff
@@ -28,7 +27,6 @@ public static class JsonSerializableExtensions
             Formatting = Formatting.Indented,
             ObjectCreationHandling = ObjectCreationHandling.Replace,
             DefaultValueHandling = DefaultValueHandling.IgnoreAndPopulate,
-            Converters = new JsonConverter[] { new Vector2Converter() },
```

Comment:
Are you sure this isn't required? This entire code is overriding the default settings after all.

Reply:
I'll test, but presumed that the default set would still be used in this case.

Reply:
You're right, doesn't work that way. Will restore.